extensions/amp-share-tracking/0.1/amp-share-tracking.js

```diff
@@ -91,6 +96,7 @@ export class AmpShareTracking extends AMP.BaseElement {
   getIncomingFragment_() {
     dev().fine(TAG, 'getting incoming fragment');
     return viewerForDoc(this.getAmpDoc()).getFragment().then(fragment => {
+      this.originalViewerFragment_ = fragment;
```

Comment:
We should separate this into it's own method. `#getIncomingFragment` could then call it, getting essentially `viewerForDoc(this.getAmpDoc()).getFragment()` as its return.